python/tk_multi_workfiles/actions/interactive_open_action.py

```diff
@@ -236,6 +236,12 @@ def _open_workfile(self, file, env, parent_ui):
                     src_path = work_path
                     work_path = local_path
 
+        try:
+            self._app.log_metric("Open workfile")
```

Comment:
I'm wondering if we should track if the user is opening a file from a sandbox and if so where is the destination. @robblau , how do you feel about this?